diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x config.rpath -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/exaile-0.2.11/Makefile src/exaile-0.2.11/Makefile
--- origsrc/exaile-0.2.11/Makefile	2007-10-02 11:25:55.000000000 -0500
+++ src/exaile-0.2.11/Makefile	2007-12-27 00:28:45.378947900 -0600
@@ -2,7 +2,7 @@
 LIBDIR ?= /lib
 FIREFOX ?= /usr/lib/firefox
 
-all: compile mmkeys.so translations
+all: compile mmkeys.dll translations
 	@echo "Done"
 	@echo "Type: 'make install' now"
 
@@ -10,8 +10,8 @@
 	python -m compileall xl lib
 	python -O -m compileall xl lib
 
-mmkeys.so:
-	cd mmkeys && make mmkeys.so && cd .. && cp mmkeys/mmkeys.so .
+mmkeys.dll:
+	cd mmkeys && make mmkeys.dll && cd .. && cp mmkeys/mmkeys.dll .
 
 translations:
 	python po/createpot.py compile
@@ -45,7 +45,7 @@
 	install -m 644 exaile.glade $(DESTDIR)$(PREFIX)/share/exaile
 	install -m 644 equalizer.ini $(DESTDIR)$(PREFIX)/share/exaile
 	install -m 644 sql/*.sql $(DESTDIR)$(PREFIX)/share/exaile/sql
-	-install -m 644 mmkeys.so $(DESTDIR)$(PREFIX)$(LIBDIR)/exaile
+	-install -m 644 mmkeys.dll $(DESTDIR)$(PREFIX)$(LIBDIR)/exaile
 	install -m 644 images/*.png $(DESTDIR)$(PREFIX)/share/exaile/images
 	install -m 644 images/default_theme/*.png \
 	$(DESTDIR)$(PREFIX)/share/exaile/images/default_theme
@@ -80,7 +80,7 @@
 	  done
 
 clean:
-	rm -f mmkeys.so
+	rm -f mmkeys.dll
 	cd mmkeys && make clean && cd ..
 	find . -name "*.py[co]" -exec rm -f {} \;
 	find po -maxdepth 1 -regextype posix-basic -regex "po/[^.]*" -type d -exec rm -rf {} \;
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x config.rpath -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/exaile-0.2.11/mmkeys/Makefile src/exaile-0.2.11/mmkeys/Makefile
--- origsrc/exaile-0.2.11/mmkeys/Makefile	2007-09-26 10:22:26.000000000 -0500
+++ src/exaile-0.2.11/mmkeys/Makefile	2007-12-27 00:31:16.441447900 -0600
@@ -1,7 +1,7 @@
 # Copyright 2004 Joe Wreschnig. Released under the terms of the GNU GPL.
 
-mmkeys.so: build
-	cp build/lib*/mmkeys.so .
+mmkeys.dll: build
+	cp build/lib*/mmkeys.dll .
 
 build: mmkeyspy.c
 	./setup.py build
@@ -17,7 +17,7 @@
 	mv gen-tmp $@
 
 clean:
-	rm -f mmkeys.so *.o mmkeyspy.c
+	rm -f mmkeys.dll *.o mmkeyspy.c
 	rm -rf build dist
 
 distclean: clean
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x config.rpath -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/exaile-0.2.11/xl/player.py src/exaile-0.2.11/xl/player.py
--- origsrc/exaile-0.2.11/xl/player.py	2007-09-26 10:22:27.000000000 -0500
+++ src/exaile-0.2.11/xl/player.py	2007-12-27 00:31:16.441447900 -0600
@@ -845,7 +845,7 @@
     VIDEO_WIDGET.show_all()
     xlmisc.finish()
     exaile.player.playbin.get_state(timeout=50*gst.MSECOND)
-    video_sink = gst.element_factory_make('xvimagesink')
+    video_sink = gst.element_factory_make('autovideosink')
     vis = gst.element_factory_make('goom')
     exaile.player.playbin.set_property('video-sink', video_sink)
     exaile.player.playbin.set_property('vis-plugin', vis)
